src/script.js

```diff
@@ -21,11 +23,65 @@ document.addEventListener("DOMContentLoaded", function () {
   journalDescriptionInput = document.getElementById("journalDescription");
   journalDateInput = document.getElementById("journalDate");
   journalList = document.getElementById("journalList");
+
+  taskTitle = document.getElementById("taskTitle");
+  taskDetails = document.getElementById("taskDetails");
+  taskDueDate = document.getElementById("dueDate");
+
   showCalendar(currentMonth, currentYear);
+
+  // x button back to index.html
```

Comment:
Lots of commented code, I think we should remove spare code that is no longer needed

Reply:
Will be removed during code cleanup.